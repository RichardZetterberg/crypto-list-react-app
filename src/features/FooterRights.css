.footer-rights {
  padding-top: 15px;
  background-color: #c0c0c0;
  padding-bottom: 10px;
  display: inline-block;
}

.footer-rights span {
  font-size: 12px;
  font-style: normal;
  font-weight: 400;
  line-height: 19px;
  letter-spacing: 0em;
  text-align: left;
  color: #ffffff;
}