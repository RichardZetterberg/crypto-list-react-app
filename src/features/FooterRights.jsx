import React from "react";
import { Row, Col } from "antd";
import "./FooterRights.css";

const FooterRights = () => {
  return (
    <Row className="footer-rights" align="middle" justify="center">
      <Col span={24}>
          <Row justify="center">
            <span>© 2022 CRYPTO-INFO.NET</span>
          </Row>
      </Col>
      <Col span={24}>
          <Row justify="center">
            <span>All Rights Reserved © 2022 CRYPTO-INFO.NET</span>
          </Row>
      </Col>
    </Row>
  );
}

export default FooterRights;
