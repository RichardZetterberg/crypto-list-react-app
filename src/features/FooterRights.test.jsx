import { render, screen } from '@testing-library/react';
import MatchMediaMock from 'jest-matchmedia-mock';

import FooterRights from './FooterRights'

let matchMedia;

describe('Footer rights component', () => {
    beforeAll(() => {
      matchMedia = new MatchMediaMock();
    });
   
    afterEach(() => {
      matchMedia.clear();
    });

    it('Footer rights renders', () => {
      render(<FooterRights />);
    })

    it('Footer rights snapshot', () => {
        const policy = render(<FooterRights />);
        expect(policy).toMatchSnapshot();
    })
})