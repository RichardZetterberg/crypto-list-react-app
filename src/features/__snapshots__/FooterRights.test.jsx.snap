// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer rights component Footer rights snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-row ant-row-center ant-row-middle footer-rights"
      >
        <div
          class="ant-col ant-col-24"
        >
          <div
            class="ant-row ant-row-center"
          >
            <span>
              © 2022 CRYPTO-INFO.NET
            </span>
          </div>
        </div>
        <div
          class="ant-col ant-col-24"
        >
          <div
            class="ant-row ant-row-center"
          >
            <span>
              All Rights Reserved © 2022 CRYPTO-INFO.NET
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-row ant-row-center ant-row-middle footer-rights"
    >
      <div
        class="ant-col ant-col-24"
      >
        <div
          class="ant-row ant-row-center"
        >
          <span>
            © 2022 CRYPTO-INFO.NET
          </span>
        </div>
      </div>
      <div
        class="ant-col ant-col-24"
      >
        <div
          class="ant-row ant-row-center"
        >
          <span>
            All Rights Reserved © 2022 CRYPTO-INFO.NET
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
